tests/plugins/test_reactor_config.py

```diff
@@ -320,9 +320,9 @@ def test_odcs_config(self, tmpdir, default):
                 odcs:
                    signing_intents:
                    - name: release
-                     keys: [R123]
+                     keys: [R123, R234]
                    - name: beta
-                     keys: [R123, B456]
+                     keys: [R123, B456, B457]
```

Comment:
Usually, beta includes all the keys from release. Let's add `R234` here as well.

Reply:
they're kind of arbitrary strings at this point, and I liked having a test for a set of signing keys that weren't quite in release and weren't quite in beta.
but if that's not a useful thing to check, I will add R234 to beta.

Reply:
Ok